tensorflow/contrib/cmake/tf_core_framework.cmake

```diff
@@ -136,7 +204,7 @@ file(GLOB_RECURSE tf_core_framework_test_srcs
 
 list(REMOVE_ITEM tf_core_framework_srcs ${tf_core_framework_test_srcs})
 
-add_library(tf_core_framework OBJECT ${tf_core_framework_srcs})
+add_library(tf_core_framework OBJECT ${tf_core_framework_srcs} ${PROTO_TEXT_HDRS})
```

Comment:
I think the layering has (unfortunately) changed so that we need a few more source files in here. A bit of experimentation reveals that the following might be necessary:

```
    "${tensorflow_source_dir}/tensorflow/core/client/tensor_c_api.cc"
    "${tensorflow_source_dir}/tensorflow/core/common_runtime/session.cc"
    "${tensorflow_source_dir}/tensorflow/core/common_runtime/session_factory.cc"
    "${tensorflow_source_dir}/tensorflow/core/common_runtime/session_options.cc"
```

...and we'll need to exclude them from `tf_core_cpu` as well.